aries-site/src/data/structures/foundation.js

```diff
@@ -17,7 +17,23 @@ export const foundation = [
     },
     seoDescription: `Accessibility is central to the HPE Design System and 
     ensures that HPE applications are usable by as many people as possible.`,
-    sections: [],
+    sections: ['Accessibility in design', 'Accessibility in development'],
```

Comment:
would Resources not be part of the sections?

Reply:
Right now, things like 'guidance', 'resources', 'variants' will be too generic for our search and end up polluting the results. Holding off for now.

Reply:
That's a good point!